lib/methods.js

```diff
@@ -0,0 +1,129 @@
+module.exports = self => {
+  return {
+    async export(req, manager) {
+      if (!req.user) {
+        throw self.apos.error('forbidden');
+      }
+
+      const ids = self.apos.launder.ids(req.query._ids);
+      const relatedTypes = self.apos.launder.strings(req.query.relatedTypes);
+
+      const allIds = self.getAllModesIds(ids);
+
+      if (!relatedTypes.length) {
+        return self.fetchActualDocs(allIds);
+      }
+
+      console.log('==> RELATED TYPED ==>');
```

Comment:
let's keep all the logs for now, they will be removed in another PR

Reply:
Why?